qiskit/pulse/builder.py

```diff
@@ -25,6 +24,12 @@
 statements. For example, below we write a simple program that :func:`play`\s
 a pulse:
 
+from qiskit.visualization import SchedStyle
+
+style = SchedStyle(figsize=(20, 10))
+
+plt.rcParams['figure.figsize'] = [15, 15]
+
```

Comment:
Is this code supposed to be a code block?  You could add a double colon in L25 or
```suggestion
.. code-block:: python

    from qiskit.visualization import SchedStyle
    style = SchedStyle(figsize=(20, 10))
    plt.rcParams['figure.figsize'] = [15, 15]
```